### PR TITLE
Deploy API continuously

### LINKED ISSUE #106: Deploy the prerelease API continuously
In order to ease testing, deploy continuously.

### PR DESCRIPTION
**Pull request by fpagnoux**
Connected to #106 

Once merged:
- [ ] Replace the hardcoded `deploy.sh` on the server by a symlink.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,7 +20,13 @@ nosetests
...
-COUNTRY_PACKAGE=openfisca_france FLASK_APP=openfisca_web_api/server.py flask run --with-threads
+COUNTRY_PACKAGE=openfisca_france gunicorn "openfisca_web_api.app:create_app()" --workers 3
 ```
 
-The `--with-threads` is necessary to avoid [this issue](https://github.com/corydolphin/flask-cors/issues/147#issuecomment-289539799). Without it, AJAX requests from Chrome sometimes take more than 20s to process.
+The `--workers k` (with `k >= 3`) option is necessary to avoid [this issue](http://stackoverflow.com/questions/11150343/slow-requests-on-local-flask-server). Without it, AJAX requests from Chrome sometimes take more than 20s to process.
+
+## Deploy
+
+```
```
Comment (cbenz) [inline README.md]: Missing `sh`

File: prod/deploy.sh [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -ex
+
+source /home/openfisca/virtualenvs/new-api/bin/activate
```
Comment (cbenz) [inline prod/deploy.sh]: Use `$HOME` like so: `source $HOME/virtualenvs/new-api/bin/activate`

File: prod/deploy.sh [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+cd /home/openfisca/new-api/openfisca-web-api
```
Comment (cbenz) [inline prod/deploy.sh]: Use `$HOME`

File: prod/deploy.sh [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+git fetch origin rewrite
+git checkout origin/rewrite --force
+pip install --editable . --upgrade
+pip install openfisca_france --upgrade
+sudo systemctl restart openfisca-web-api-new.service
```
Comment (cbenz) [inline prod/deploy.sh]: Add a comment to indicate this specific command is allowed using `visudo`

File: prod/openfisca-web-api.service [hunk 5]
```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=OpenFisca Web API
```
Comment (cbenz) [inline prod/openfisca-web-api.service]: Rename "OpenFisca new Web API"

Comment (fpagnoux) [inline prod/openfisca-web-api.service]: This was on purpose. As @MattiSG [mentioned earlier](https://github.com/openfisca/openfisca-web-api/pull/105#discussion_r107103222), I don't think we should be writing "new" api everywhere.

File: README.md [hunk 6]
```diff
@@ -18,7 +20,13 @@ nosetests
 ## Run
 
 ```sh
-COUNTRY_PACKAGE=openfisca_france FLASK_APP=openfisca_web_api/server.py flask run --with-threads
+COUNTRY_PACKAGE=openfisca_france gunicorn "openfisca_web_api.app:create_app()" --workers 3
```
Comment (cbenz) [inline README.md]: I think you should add `--bind localhost:5000` to match the `development.js` config of Legislation Explorer.

File: prod/deploy.sh (hunk 2 above)
Comment (fpagnoux) [inline prod/deploy.sh]: Arg no, the deploy script is run by a deployment user, who is not `openfisca`. We cannot use `$HOME`.

**Pull request merged**